package.json

```diff
@@ -36,26 +36,54 @@
     "@fontsource/lato": "^4.5.10",
     "@fontsource/pt-sans": "^4.5.11",
     "@fontsource/pt-serif": "^4.5.11",
+    "@manuscripts/assets": "^0.6.1",
+    "@manuscripts/body-editor": "^1.1.0",
+    "@manuscripts/comment-editor": "^1.0.1",
+    "@manuscripts/data": "^0.1.11",
+    "@manuscripts/json-schema": "^1.0.0",
+    "@manuscripts/library": "^1.0.2",
+    "@manuscripts/quarterback-types": "^0.2.2",
+    "@manuscripts/requirements": "^1.0.3",
+    "@manuscripts/style-guide": "^1.0.7",
+    "@manuscripts/title-editor": "^1.1.0",
+    "@manuscripts/track-changes-plugin": "^1.2.0",
+    "@manuscripts/transform": "^1.1.2",
```

Comment:
Why did we move these dependencies from dev? as it was easy for us to maintain them in one place like the dashboard